src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/evosuite/Runner.kt

```diff
@@ -61,12 +61,14 @@ class Runner(
     }
 
     /**
-     * Sets up evosuite to run for a target method of the target class. This attaches a method prefix argument
+     * Sets up evosuite to run for a target method of the target class. This attaches a method descriptor argument
      * to the evosuite process.
+     *
+     * @param methodDescriptor The method descriptor of the method under test
      */
-    fun forMethod(method: String): Runner {
+    fun forMethod(methodDescriptor: String): Runner {
         command =
```

Comment:
What is the key difference between you using `method` and `methodDescriptor`?

Reply:
I think that `method` is a method prefix, while `methodDescriptor` is the entire method signature.

Reply:
what is the key difference that made you switch

Reply:
With method prefix, EvoSuite will generate tests for all the overloads of the given method name. With descriptor - only for the selected one, since the method signature is unique